clients/rospy/src/rospy/topics.py

```diff
@@ -189,7 +193,7 @@ class Poller(object):
     """
     def __init__(self):
         try:
-            self.poller = select.poll()
+            self.poller = select.epoll()
```

Comment:
The Poller class in topics.py provides a unified interface for polling on all three platforms. I cleaned up the code a bit (48f4f0a) and added a fallback to select.poll for Linux <  2.5.44. I did not touch the code for OS X and Windows. So regarding the compatibility I'd say that this PR does not affect the compatibility.